src/features/DebugSession.ts

```diff
@@ -51,6 +54,7 @@ export class DebugSessionFeature implements IFeature {
                 if (currentDocument.isUntitled) {
                     if (currentDocument.languageId === 'powershell') {
                         config.script = currentDocument.uri.toString();
+                        config.cwd = vscode.workspace.rootPath;
```

Comment:
Actually now that I think about it, this line is probably not needed anymore because the || statement above should cover that case.  Leftover from a previous attempt